### PR TITLE
Unify drop shadow styles

### PR DESCRIPTION
**Pull request by tofumatt**
Fixes https://github.com/mozilla/addons.mozilla.org-mod/issues/2

### Before

ss1

### After

ss2

Pretty sure this is what we're after, but if there are other places I should make sure this was applied, please let me know @pwalm.

### CODE REVIEW

**Comment by muffinresearch** [comment]
The "after" looks weird to me. What's with all the green lines is that expected or is just the screen-shot that's borked?

**Comment by pwalm** [comment]
Looks good, other than the aforementioned green line at the bottom.

———

Philip Walmsley
Senior Visual Designer, Firefox UX
ping: @pwalm
tweet: @pwalmsley
txt: 1 604 339 5941
tumbl: http://pwalm.tumblr.com

**Comment by muffinresearch** [comment]
@pwalm is there a reference mock for this?

**Comment by pwalm** [comment]
Nope, we were just free styling it with this re-skin. Now that I look closer, I'd say "before" is the ideal state. Didn't notice the wonky drop shadow at the top of the dropdown.

**Comment by tofumatt** [comment]
The green line was part of the style mentioned in https://github.com/mozilla/addons.mozilla.org-mod/issues/2, so I think it was intentional.

@pwalm: are you saying we should nix this and go back to the previous/original drop-shadow?

**Comment by muffinresearch** [comment]
What makes it look odd is that the arrow ^ still has a different shadow on it too. Sources of light to create shadows should ideally be consistent otherwise it looks strange.

@pwalm Here's a couple of jsbins in-case it's easier to experiment:

New styles as per the PR:
http://jsbin.com/zapivi/edit?css,output

Old styles (albeit with very small tweaks to the arrow shadow to make it look more consistent with the main overlay shadow).

http://jsbin.com/yopuve/edit?css,output

Sometime I find jsbin doesn't output anything. If that happens try re-selecting "less" here:

js_bin_-_collaborative_javascript_debugging

**Comment by pwalm** [comment]
Old style with the tweaks. Looks more uniform:
http://jsbin.com/yopuve/edit?css,output

**Comment by tofumatt** [comment]
Cool. I've moved to that style. Screenshots:

screenshot 2016-02-26 23 48 29
screenshot 2016-02-26 23 48 24

Quick r? @muffinresearch

File: static/css/restyle.less [hunk 1]
```diff
@@ -28,6 +29,17 @@
     src: local('Open Sans Italic'), local('OpenSans-Italic'), url('../../css/impala/fonts/OpenSans-Italic.ttf') format('ttf');
 }
 
+// Mixins
+.drop-shadow {
```
Comment (muffinresearch) [inline static/css/restyle.less]: Do you think this will be re-used? I'm not sure the mixin is necessary. Also having the `&:after` in it seems a bit odd to me.

Comment (tofumatt) [inline static/css/restyle.less]: I guess not, but I like the notion of unifying styles and using a constant mix-in. But I can take it out, given it’s not really re-used, fair enough.

**Comment by tofumatt** [comment]
Okay, removed the mixin. The `&:after` bit was weird, agreed.

File: static/css/restyle.less [hunk 2]
```diff
@@ -672,8 +674,9 @@ button.good {
 }
 
 .install-button > .button.add.installer {
+    box-shadow: 0 0 4px rgba(0, 0, 0, 0.4);
+
     background: #57BD35;
-    box-shadow: 0 2px 0 0 rgba(0,0,0,0.10), inset 0 -2px 0 0 rgba(56,136,30,0.50);
```
Comment (muffinresearch) [inline static/css/restyle.less]: Does this bit need to be changed too? It doesn't look to be related to the overlay.

Comment (tofumatt) [inline static/css/restyle.less]: No, I don't know how that happened. Blarg.

**Comment by tofumatt** [comment]
Okay, should be all fixed.

**Comment by muffinresearch** [comment]
r+
